tests/config.bats

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bats
+load test-helper
+
+@test "$clinom config" {
+  skip "Run $clinom config with the [update|add|delete|clear] subcommands"
+  run "${clicmd}" config
+  [ "$status" -eq 0 ]
+}
+
+@test "$clinom config update" {
+  run "${clicmd}" config update abc 123
+  if [[ "$output" == *"Error"* ]]; then
+    skip "Missing varname or varvalue"
+  fi
+  assert_success
+}
+
+@test "$clinom config add" {
+  run "${clicmd}" config add abc 123
+  if [[ "$output" == *"Error"* ]]; then
+    skip "Missing varname or varvalue"
+  fi
```

Comment:
This test could create a fake config variable like an apitoken for Gitter/Discord/Slack.